File: prograimg_python_basic/Exercises/conditional_statements/conditional_statements_-_lab/area_of_figures.py
from math import pi

figure = str(input())

if figure == "square":
    square = float(input())
    size_square = square * square
    print("{:.3f}".format(size_square))
elif figure == "rectangle":
    a = float(input())
    b = float(input())
    size_rectangle = a * b
    print("{:.3f}".format(size_rectangle))
elif figure == "circle":
    radius = float(input())
    size_circle = pi * (radius * radius)
    print("{:.3f}".format(size_circle))
elif figure == "triangle":
    a = float(input())
    b = float(input())
    size_triangle = a * b / 2
    print("{:.3f}".format(size_triangle))

File: JS/JavaScript/JS-Basics/ConditionalStatementsLab/AreaOfFigures.js
function area(inputs) {

    let figure = String(inputs[0])

    if (figure == 'square') {
        let length = parseFloat(inputs[1])

        console.log((length ** 2).toFixed(3))
    }
    
    else if (figure == 'rectangle') {
        let length = parseFloat(inputs[1])
        let side = parseFloat(inputs[2])
        console.log((length * side).toFixed(3))
    }
    else if (figure == 'circle') {
        let radius = parseFloat(inputs[1])
        console.log((Math.PI * (radius ** 2)).toFixed(3))
    }
    else if (figure == 'triangle'){
        let length = parseFloat(inputs[1])
        let length_two = parseFloat(inputs[2])
        console.log((length * length_two / 2).toFixed(3))
    }
    ;

    
}


area(["square", "5"])
area((["rectangle","7","2.5"]))
area((["circle","6"]))
area((["triangle","4.5","20"]))